client/kv.go

```diff
@@ -0,0 +1,168 @@
+package client
+
+import (
+	"errors"
+	"fmt"
+
+	"github.com/google/uuid"
+	xlineapi "github.com/xline-kv/go-xline/api/xline"
+)
+
+// / Client for KV operations.
+type kvClient struct {
+	// Name of the kv client, which will be used in CURP propose id generation
+	name string
+	// The client running the CURP protocol, communicate with all servers.
+	curpClient curpClient
+	// The auth token
+	token string
+}
+
+// Client for KV operations.
+func newKvClient(name string, curpClient curpClient, token string) kvClient {
+	return kvClient{name: name, curpClient: curpClient, token: token}
+}
+
+// Get a range of keys from the store
+func (c kvClient) Range(request *xlineapi.RangeRequest) (*xlineapi.RangeResponse, error) {
+	keyRange := []*xlineapi.KeyRange{
+		{
+			Key:      request.Key,
+			RangeEnd: request.RangeEnd,
+		},
+	}
+	requestWithToken := xlineapi.RequestWithToken{
+		Token: &c.token,
+		RequestWrapper: &xlineapi.RequestWithToken_RangeRequest{
+			RangeRequest: request,
+		},
+	}
+	proposeId := c.generateProposeId()
+	cmd := xlineapi.Command{Keys: keyRange, Request: &requestWithToken, ProposeId: proposeId}
+	fmt.Println("temp")
```

Comment:
Please remove this debug print

Reply:
Oops, sorry I missed that! I've updated it as you recommended.